### PR TITLE
Fix jQuery check issue

### LINKED ISSUE #24: jQuery check causes error to throw
### Expected Behavior
Constructor runs as expected

### Actual Behavior
`Uncaught TypeError: Expecting a function in instanceof check, but got undefined`

### Steps to Reproduce
1. remove jQuery from your window: `window.jQuery = undefined`
2. create a new player with a non-jquery element `new Vimeo.Player(getPlayerElement())`
3. watch the console blow up

### PR DESCRIPTION
**Pull request by BelfordZ**
Fixes #24

### CODE REVIEW

File: src/player.js [hunk 1]
```diff
@@ -23,8 +23,8 @@ class Player {
      */
     constructor(element, options = {}) {
         /* global jQuery */
-        if ('jQuery' in window && element instanceof jQuery) {
```
Comment (bdougherty) [inline src/player.js]: I think the simpler change here is to also check that jQuery is not `undefined`:

```js
if ('jQuery' in window && typeof window.jQuery !== 'undefined' && element instanceof jQuery) {
```

Comment (BelfordZ) [inline src/player.js]: Perhaps, but checking jQuery in window is just pointless. Its bug prone and just shouldn't be used.

**Comment by bdougherty** [comment]
Can you add a test for this?

**Comment by BelfordZ** [comment]
@bdougherty updated with tests. Thanks for helping get this resolved quickly 👍

File: package.json [hunk 2]
```diff
@@ -59,6 +59,8 @@
     "babel-plugin-transform-es2015-modules-commonjs": "6.8.0",
     "babel-plugin-transform-runtime": "6.9.0",
     "babel-preset-es2015-loose-rollup": "7.0.0",
+    "babel-register": "6.9.0",
```
Comment (BelfordZ) [inline package.json]: these deps were missing and prevented me from testing

**Comment by BelfordZ** [comment]
@bdougherty also, I generated a build. Would you like me to remove that?

File: package.json (hunk 2 above)
Comment (bdougherty) [inline package.json]: What version of node/npm do you have? I did a fresh install and both were installed for me.

**Comment by bdougherty** [comment]
@BelfordZ yes, please remove it. It's automatically updated when I generate a new version.

**Comment by BelfordZ** [comment]
@bdougherty good to go!

**Comment by BelfordZ** [comment]
@bdougherty Anything else you need? Really need this fix!

**Comment by bdougherty** [comment]
Merged! Thanks @BelfordZ

**Pull request merged**